src/main/java/com/epam/tc/hw3/DifferentElementsPage.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.tc.hw3;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DifferentElementsPage {
+
+    @FindBy(className = "label-checkbox")
+    private List<WebElement> checkBoxes;
+
+    @FindBy(className = "label-radio")
+    private List<WebElement> checkRadio;
+
+    @FindBy(className = "colors")
+    private WebElement dropdownWithColors;
+
+    @FindBy(tagName = "option")
+    private List<WebElement> colorsInDropdown;
+
+    @FindBy(css = "ul[class='panel-body-list logs']")
+    private WebElement logRows;
+
+
+
+    public DifferentElementsPage selectWaterCheckbox() {
```

Comment:
one of the requirements of hw3 is using Void Page Object pattern in Page Object.
Fix all methods, which returning the page instance